cypress/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
PTAL https://github.com/red-hat-storage/odf-console/pull/102#discussion_r787738206

Reply:
Since we don't immediately have a need for `Object.values`, we can technically remove this `.tsconfig`, but nonetheless, we will need to add it later when migrating tests. PLMK WDYT.